src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.ts

```diff
@@ -242,4 +289,32 @@ export class ServiceDaemonListComponent implements OnInit, OnChanges, AfterViewI
   trackByFn(_index: any, item: any) {
     return item.created;
   }
+
+  updateSelection(selection: CdTableSelection) {
+    this.selection = selection;
+  }
+
+  daemonAction(actionType: string) {
+    if (this.selection.first()?.daemon_type !== 'mon' &&
+      this.selection.first()?.daemon_type !== 'mgr') {
```

Comment:
Instead of doing this why not simply disabling the actions when 'mon' and 'mgr' is selected?

Reply:
same comment as below